### DISCUSSION TITLE
Introductions: Welcome Bruce!

### DISCUSSION THREAD

**Discussion by sspickle** [UINDY-SWEN-200/flaskGlowscriptHost#8] [General]
Hello Team,

I've invited @BruceSherwood to join the development team for this project. Bruce has been the motivating force behind WebVPython and VPython since the beginning. Together with students and colleagues, Bruce started [WebVPython in 2011](https://www.glowscript.org/docs/VPythonDocs/history.html) based on the earlier [VPython](https://vpython.org/contents/history.html) project started in the early 2000s. I'm guessing Bruce will not contribute code directly to this project, but he is the keeper of all arcane knowledge about WebVPython, and I'm sure he would welcome questions, if you have any, about the context of WebVPython or why things are currently as they are. If you do choose to contact Bruce, please be sure to copy me on any messages!

Welcome, Bruce to our class project!

Take care,
-Steve

**Comment by BruceSherwood** [comment]
Here is a history of VPython:

    https://brucesherwood.net/?p=136

Bruce

P.S. You'll see that Steve made major contributions to VPython 6. He has
also made major contributions to the current VPython, especially the Web
VPython flavor.
